scripts/vcpkg_ports/raja/portfile.cmake

```diff
@@ -1,8 +1,8 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO llnl/raja
-    REF v2024.07.0
-    SHA512 7337d21dad05682c87145d23d09ae1c4cf627c21386e65a96f55beb0969cb2ef3cfaa38d5eaf5322ba04724707c6d993509b4ea6c2b679db6a3c4e5ac1d9b441
+    REF 85c58eb7d25ac5a7ff1c6a3cc6bb74bf8351e36e # HEAD of bugfix/dayton8/win32 branch; a bit past v2024.07.0
+    SHA512 263346ff91c17f48fa40efdd6826331e1fecb995fef289bf997500d058c57e4dc706512baa8fc5dfc912c54939b973c56c8677a4705e261a2740c55851446341
```

Comment:
Note: This points to @adayton1's bugfix commit -- https://github.com/LLNL/RAJA/pull/1746

I can update it to the current HEAD of raja@develop now that the PR is merged (or leave it as is).

Reply:
I'm OK with either. If users need this, I can tag a patch release for RAJA because Mike C. won't build any TPL sets that don't point at releases.

Reply:
Thanks, but I don't think it's necessary.

Axom's only using this on Windows within our custom vcpkg port for RAJA, so I think it's ok to use a custom commit a bit past the RAJA release.